signer/openrpc.json

```diff
@@ -21,18 +23,18 @@
         "name": "timestamp",
         "description": "Seconds since Jan 01 1970 (UTC)",
         "schema": {
-          "type": "integer",
-          "format": "int64"
```

Comment:
why remove the int64? 

If anything, dropping it to int16 would be better.

Reply:
because its not a format JSON Schema supports. number is just any number, no bounds, its up to the language.

theres a big rabbit hole if you want to go down it:
https://github.com/json-schema-org/json-schema-spec/issues/574

Reply:
or you can do `minimum: 0` and `maximum: ` if we want to go that route